lib/src/shared_model/shared_model.dart

```diff
@@ -23,6 +23,7 @@ export 'catalog_object_type.dart';
 export 'catalog_query.dart';
 export 'catalog_query_exact.dart';
 export 'catalog_query_items_for_item_option_values_query.dart';
+export 'catalog_query_items_for_item_option_values_query.dart';
```

Comment:
Looks like a little oops

Reply:
hmmm, strange.
The linter was telling my that is wasn't exported in shared_model.dart
So, I exported it and low and behold there are two.
However, when I remove one of them, the linter freaks out again.
Turns out if a change the order of parameter CatalogQuery the linter stops screaming at me... 
STRANGE!!!

Reply:
Should be resolved now.